app/src/main/res/layout/subscriber_repositories_item.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/subscriber_repositories_item_height"
+    android:orientation="horizontal"
+    android:tag="@string/subscriber_details_layout_tag">
+
+    <LinearLayout
```

Comment:
This linear layouts are useless. You can easily remove it and move the properties to the textviews